1.7/administration/storage.md

```diff
@@ -0,0 +1,113 @@
+---
+post_title: Storage
+menu_order: 10
+---
+
+### Mount Disk Resources
+
+#### Overview
+
+Mesos Volume Mounts give services the ability to reserve dedicated disk space throughout the cluster. This is particularly useful for stateful services like Kafka and Cassandra.
+
+In DC/OS it is possible to configure [Mesos Mount disk resources][1] across the cluster by simply mounting storage resources on Agents using a well-known path. When a DC/OS agent starts for the first time it will scan for volumes matching the pattern `/dcos/volumeN`, where N is an integer, and automatically configure the Mesos Agent to offer these disk resources to other services.
+
+##### Example using loopback device
+
+The example below shows how to add a disk resource to a DC/OS agent post-install on a running cluster. The same steps can be used pre-install, but in that scenario it is not necessary to stop services or clear the Mesos Agent state. All commands below should be run as root.
+
+***Warning:*** This will terminate any running tasks or services on the node.
+
+1. Connect to an agent in the cluster with SSH
+
+2. Examine the current Mesos Agent resource state
+
+  Note there are no references yet for `/dcos/volume0`.
+
+	```
+	root@dcos-agent-public:~# cat /var/lib/dcos/mesos-resources
+	# Generated by make_disk_resources.py on 2016-05-05 17:04:29.868595
+	#
+
+	MESOS_RESOURCES='[{"ranges": {"range": [{"end": 21, "begin": 1}, {"end": 5050, "begin": 23}, {"end": 32000, "begin": 5052}]}, "type": "RANGES", "name": "ports"}, {"role": "*", "type": "SCALAR", "name": "disk", "scalar": {"value": 47540}}]'
+	```
+
+3. Stop the Mesos Agent
```

Comment:
I thought about that, but we can't combine without making it into a little subscript or by swallowing systemctl errors.